kubeflow/core/iap.libsonnet

```diff
@@ -251,6 +251,78 @@
     },  // deploy
     deploy:: deploy,
 
+    // Run the process to update the backend service
+    local backendUpdater = {
```

Comment:
Should we run this as a StatefulSet  or Deployment?

It could be desirable; if we periodically rerun the commands to update the backend. This way if someone messes with the backend it will eventually be reset to the desired state.

Reply:
Updated as Statefulset